folium/map.py

```diff
@@ -246,6 +249,26 @@ def _repr_html_(self, **kwargs):
             out = self._parent._repr_html_(**kwargs)
         return out
 
+    def _repr_png_(self):
+        """Displays the PNG Map in a Jupyter notebook."""
+        # https://github.com/ipython/ipython/issues/7899#issuecomment-76541102
```

Comment:
The notebook will run all `_repr_*_` methods and, b/c this one can be slow, we don't really want it to be called without the user request. Not sure yet if this should be a `_repr_png_` of just a `to_png()` method.